### ISSUE TITLE
Intended behavior of layering (z-index?) of price lines relative to candlesticks data? How do I have price lines show above the candlesticks?

### ISSUE THREAD

**Issue by jakemayfield** [tradingview/lightweight-charts#1120]
I classified this issue as "Other" in the event that it's not a bug and I'm going about this the incorrect way.

What is the intended behavior of the layering of the price lines relative to the candlestick data? It seems I'm unable to have the price line appear above the candlesticks data. Please see image below for what I am experiencing.

<img width="1018" alt="Screen Shot 2022-07-14 at 6 15 51 PM" src="https://user-images.githubusercontent.com/14812609/179121784-eccd6799-d2cb-4dff-86e3-42bdbfab824c.png">

I tried calling the createPriceLine() and setData() methods on the candlestick series both before and after each other but was unable to achieve a different outcome in the ordering in the layering of elements in the chart. The candlesticks data always appears above or on top of the price lines, as shown in the image.

Am I missing something?

These price lines are attached to the candlestick series.

I love the library, by the way. 

Many thanks!

**Comment by jakemayfield** [comment]
Duplicate of https://github.com/tradingview/lightweight-charts/issues/684 and https://github.com/tradingview/lightweight-charts/pull/806?

Are we still waiting on the next major version (4.0.0) to drop before seeing this fix? I'm using version 3.8.0 (latest).

**Comment by SlicedSilver** [comment]
Yes, this is fixed in the next major release (4.0.0). If you need the fix before it is released then you can clone this repo and build the library off the `master` branch. You could then use the pre-release version within the `dist` folder.

**Comment by SlicedSilver** [comment]
This issue has been resolved with the [`v4.0.0`](https://github.com/tradingview/lightweight-charts/releases/tag/v4.0.0) release.
